app/services/fetch_coronavirus_statistics_service.rb

```diff
@@ -105,15 +108,53 @@ def latest_vaccinations(data)
 
   def latest_admissions(data)
     latest_admissions = data.find { |d| d["hospitalAdmissions"] }
-
     return {} unless latest_admissions
 
+    daily_admissions = daily_admissions(latest_admissions)
+    weekly_admissions = weekly_admissions(latest_admissions, data)
+
+    daily_admissions.merge!(weekly_admissions)
+  end
+
+  def daily_admissions(latest_admissions)
     {
       hospital_admissions: latest_admissions["hospitalAdmissions"],
       hospital_admissions_date: Date.parse(latest_admissions["date"]),
     }
   end
 
+  def weekly_admissions(latest_admissions, data)
+    weeks_with_admissions = []
+    data.each do |day_admissions|
```

Comment:
Can you expand the commit message slightly to say that `nil` would be created both by a lack of data or a `false` on the line below and we want one but not the other please (discussed offline (: )